addons/istio/disable

```diff
@@ -4,9 +4,13 @@ set -e
 
 source $SNAP/actions/common/utils.sh
 
+KUBECTL="$SNAP/microk8s-kubectl.wrapper"
+
 echo "Disabling Istio"
 
-run_with_sudo "${SNAP_DATA}/bin/istioctl" -c "${SNAP_DATA}/credentials/client.config" x uninstall --purge -y
+run_with_sudo "${SNAP_DATA}/bin/istioctl" -c "${SNAP_DATA}/credentials/client.config" uninstall --purge -y
```

Comment:
I'm a bit concerned that the removal of the `x` will break backwards compatibility with pre-1.15 versions of `istioctl`.

I will defer to @ktsakalozos @bschimke95 to decide whether that's something we even care about since istio 1.15 has been EoL for over a year now.

Full context:

[The upstream PR which first graduated  `istio uninstall`](https://github.com/istio/istio/pull/40362) (and thus added the subcommand without the `x`) was first [released in 1.15.0](https://istio.io/latest/news/releases/1.15.x/announcing-1.15/#istioctl-uninstall)

[The PR which "fully graduated" it](https://github.com/istio/istio/pull/45640) (and thus removed `istioctl x uninstall`) was [first shipped in 1.19.0](https://istio.io/latest/news/releases/1.19.x/announcing-1.19/change-notes/#istioctl).

Reply:
Let's just add a check for versions before 1.15 to be on the safe side.
Something like:
```
if version < 1.15 {
  istioctl x uninstall ...
} else {
  istioctl uninstall ...
}
```